src/Mod/Part/Gui/Command.cpp

```diff
@@ -2128,8 +2127,8 @@ CmdBoxSelection::CmdBoxSelection()
 {
     sAppModule    = "Part";
     sGroup        = QT_TR_NOOP("Part");
-    sMenuText     = QT_TR_NOOP("Box selection");
-    sToolTipText  = QT_TR_NOOP("Box selection");
+    sMenuText     = QT_TR_NOOP("Box Selection");
+    sToolTipText  = QT_TR_NOOP("Box selects elements in the 3D view");

```

Comment:
```suggestion
    sToolTipText  = QT_TR_NOOP("Selects elements in the 3D view using a box selection");
```
To be consistent with the verb-first style we have